.storybook/main.js

```diff
@@ -1,52 +1,37 @@
 const path = require('path');
 
 module.exports = {
-  stories: ['../src/**/*.stories.tsx'],
-  addons: [
-    '@storybook/addon-essentials',
-    '@storybook/addon-links',
-    '@storybook/addon-storysource',
-    '@storybook/addon-a11y/register',
+  "stories": [
+    "../src/**/*.stories.tsx"
+  ],
+  "addons": [{
+      name: '@storybook/preset-scss',
+      options: {
+        cssLoaderOptions: {
+          modules: {
+            auto: true
+          }
+        }
+      }
+    },
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-a11y"
   ],
   typescript: {
-    reactDocgen: false,
+    reactDocgen: false
+  },
+  "core": {
+    "builder": "webpack5"
   },
   webpackFinal: async (config) => {
```

Comment:
good riddance, didn't like custom webpack setups in the first place. 

Thank you!